### PR TITLE
add cronoscan

### PR DESCRIPTION
**Pull request by wujunze**
add cronoscan

### CODE REVIEW

**Comment by wujunze** [comment]
Hello  @hasparus  Please Review

**Comment by hasparus** [comment]
cc @krzkaczor

File: packages/ethereum-viewer/src/explorer/networks.ts [hunk 1]
```diff
@@ -89,4 +94,7 @@ export const explorerApiKeys: Record<ApiName, string> = {
 
   snowtrace: SNOWTRACE_KEY,
   "testnet.snowtrace": SNOWTRACE_KEY,
+  
+   cronoscan: FTMSCAN_KEY,
+  "testnet.cronoscan": FTMSCAN_KEY,
```
[the remark this answers is not in the document]
Comment (wujunze) [reply packages/ethereum-viewer/src/explorer/networks.ts:99]: Thanks

**Comment by wujunze** [comment]
Hello   @hasparus @krzkaczor    fixed   
Please Review

**Comment by krzkaczor** [comment]
Note: I updated the API key and removed not working testnet.

**Comment by krzkaczor** [comment]
Thanks for the contribution @wujunze. It should be deployed in few minutes. Please test it.

**Comment by wujunze** [comment]
Hello  @krzkaczor @hasparus    
            Github Actions Build error    https://github.com/dethcrypto/dethcode/runs/6311920040?check_suite_focus=true 
           Please Fix     Thanks

**Comment by krzkaczor** [comment]
@wujunze fix'd

**Comment by wujunze** [comment]
Thanks @krzkaczor

**Pull request merged**